tests/restarting/from_7.2.0/DrUpgradeRestart-1.toml

```diff
@@ -0,0 +1,28 @@
+[configuration]
+extraDatabaseMode = "Local"
+disableEncryption = true
+# DR is not currently supported in required tenant mode
+tenantModes = ['disabled', 'optional']
+
+[[test]]
+testTitle = "DrUpgrade"
+clearAfterTest = false
+simBackupAgents = "BackupToDB"
+
+[[test.workload]]
```

Comment:
Typically the `[[test.workload]]` sections are indented in the test specs. I'm also not 100% sure whether the `tenantModes` option is supported in version 7.2, so we should confirm that.

Reply:
It's supported. I encountered this test use tenant mode = required

Reply:
Yeah, `allowDefaultTenant = false` should be used in 71.2 and 7.2